Code/Tools/SceneAPI/SceneCore/DataTypes/Rules/IReadOnlyRule.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
```

Comment:
Nit: May worth moving this line after the copyright comment.

Reply:
Done, and I updated a bunch of other ones to prevent this copy & paste error from happening again.